bioinput.py

```diff
@@ -1,10 +1,32 @@
-class Bioinput():
-    """
+import pandas as pd
+
+
+class Bioinput:
+    """Input Class
 
     """
 
     def __init__(self, file="ecg_data.csv"):
         self.file = file
 
     def read_input(self):
-        print("BME590")
\ No newline at end of file
+        """Read in a CSV file and convert it into a DataFrame with the columns being time and voltage"
+
+        :return: An DataFrame for the time and voltage ecg data
+        """
+
+        try:
+            ecg_dataframe = pd.read_csv(self.file, names=['time', 'voltage'], skiprows=1)
+            if len(ecg_dataframe) == 0:
+                print("No data found")
+                return ecg_dataframe
```

Comment:
If the length of `ecg_dataframe` is 0, then maybe set to `None` and handle that downstream?